src/shapes/text.class.js

```diff
@@ -373,6 +373,7 @@
         this.setPathInfo();
       }
       this.__skipDimension = false;
+      this.styles = this._stylesFromArray();
```

Comment:
I think it's best to pass the style array and not set it on the instance
It doesn't seem safe